.github/workflows/review.yml

```diff
@@ -12,6 +12,24 @@ on:
       - applications/**/*.json
 
 jobs:
+  setup:
+    name: Setup
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checklist
+        uses: elementary/action-appcenter-review-checklist@v1.0.0
+        with:
+          body: |
+            ## Review Checklist
+
+            - [ ] App opens
+            - [ ] Does what it says
+            - [ ] Matches description in AppData
+            - [ ] Matches screenshot
+            - [ ] AppData custom colors are contrasty
+            - [ ] Name is unique and non-confusing
```

Comment:
The markdown is configurable, but this is the list that Cass gave so I think it's a pretty good starting point. Easily editable down the road.

Reply:
Yeah I was thinking the same. I'm probably missing things but we can iterate as we review things and learn what we need to explicitly check.